save-orchestrator/build.gradle.kts

```diff
@@ -19,17 +19,15 @@ tasks.withType<KotlinCompile> {
     }
 }
 
-tasks.getByName("processResources").finalizedBy("downloadSaveCli")
+// if required, file can be provided manually
 val saveCliVersion = getSaveCliVersion()
-tasks.register<Download>("downloadSaveCli") {
-    dependsOn("processResources")
-    // if required, file can be provided manually
-    if (file("$buildDir/resources/main/save-$saveCliVersion-linuxX64.kexe").exists()) {
-        src(file("$buildDir/resources/main/save-$saveCliVersion-linuxX64.kexe").toURI().toURL())
-    } else {
+if (!file("$buildDir/resources/main/save-$saveCliVersion-linuxX64.kexe").exists()) {
```

Comment:
What if file exists?

Reply:
Then it will be included in the image, no need to download anything